src/app/debug/table/table.component.ts

```diff
@@ -189,15 +197,14 @@ export class TableComponent implements OnInit, OnDestroy {
 
   loadData(): void {
     this.loadReportInProgressThreshold();
-    this.httpService.getViews().subscribe((views) => {
+    this.httpService.getViews().subscribe((views: Record<string, View>) => {
       if (Object.keys(this.viewSettings.currentView).length > 0) {
         this.debugReportService.changeView(this.viewSettings.currentView);
       } else {
         this.viewSettings.views = views;
         this.calculateViewDropDownWidth();
-        this.viewSettings.currentViewName = Object.keys(this.viewSettings.views).find(
-          (view) => this.viewSettings.views[view].defaultView,
-        );
+        this.viewSettings.currentViewName =
+          Object.keys(this.viewSettings.views).find((view: string) => this.viewSettings.views[view].defaultView) || '';
```

Comment:
I prefer the `??` operator to the `||` operator here, using `||` will not work as expect in some case (like with booleans) while `??` does. For more info see [Nullish coalescing operator](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Nullish_coalescing)